models/OrderDepositTransaction.go

```diff
@@ -0,0 +1,74 @@
+package models
+
+import (
+	"github.com/Sirupsen/logrus"
+	"github.com/delta/dalal-street-server/utils"
+)
+
+type OrderDepositTransaction struct {
+	TransactionId uint32 `gorm:"column:transactionId;not null" json:"transaction_id"`
+	OrderId       uint32 `gorm:"column:orderId;not null" json:"bid_id"`
+	IsAsk         bool   `gorm:"column:isAsk;not null" json:"is_ask"`
+	CreatedAt     string `gorm:"column:createdAt;not null" json:"created_at"`
+}
+
+func (OrderDepositTransaction) TableName() string {
+	return "OrderDepositTransactions"
+}
+
+// GetOrderDepositTransactionRef returns a reference of
+func GetOrderDepositTransactionRef(transactionID, orderID uint32, isAsk bool) *OrderDepositTransaction {
+	return &OrderDepositTransaction{
+		TransactionId: transactionID,
+		OrderId:       orderID,
+		IsAsk:         isAsk,
+		CreatedAt:     utils.GetCurrentTimeISO8601(),
+	}
+}
+
+// GetPlaceOrderTransactionDetails returns price and quantity at which PlaceOrderTransaction was created
+func GetPlaceOrderTransactionDetails(orderID uint32, isAsk bool) (int64, int64, error) {
```

Comment:
Does this need to be exported?

Reply:
User.go Line 1117. Using it while saving bid for canel order

Reply:
That's still not exported. Exporting is for across packages, not files.

Reply:
Oh. My bad. Fixed now :+1: